website/content/api-docs/secret/pki.mdx

```diff
@@ -3595,11 +3595,27 @@ based on this parameter.
 - `not_before_duration` `(duration: "30s")` - Specifies the duration by which to
   backdate the NotBefore property. This value has no impact in the validity period
   of the requested certificate, specified in the `ttl` field.
+  
+- `not_before_bound` `(string: "permit")` - Specifies if and how `not_before` can be 
+  provided when requesting a certificate. Defaults to `permit`. Valid options are :
+  
+    - `permit` to allow providing `not_before`.
+    - `duration` to allow providing `not_before`within the bounds of `not_before_duration`.
```

Comment:
```suggestion
    - `duration` to allow providing `not_before` within the bounds of `not_before_duration`.
```

Reply:
Fixed in https://github.com/openbao/openbao/commit/99dc0b97c4da3719ccb23216a6c3539c4d53671c